src/periphery/MgvCleaner.sol

```diff
@@ -30,7 +31,8 @@ contract MgvCleaner {
     returns (uint bal)
   {
     unchecked {
-      (uint successes,,,,) = MGV.snipesFor(outbound_tkn, inbound_tkn, targets, fillWants, msg.sender);
+      (uint successes,,,,) =
+        MgvHelpers.snipesForByVolume(address(MGV), outbound_tkn, inbound_tkn, targets, fillWants, msg.sender);
```

Comment:
Assume the MgvCleaner API was left unchanged in anticipation of it being removed?

Reply:
just minimizing changes :p